### ISSUE TITLE
cudnn_convolution-inl.h:160: Check failed: e == CUDNN_STATUS_SUCCESS (8 vs. 0) : cuDNN: CUDNN_STATUS_EXECUTION_FAILED

### ISSUE THREAD

**Issue by olojuwin** [apache/incubator-mxnet#16485]
bug
work environment：2080TI
CUDA:9.0
MXNET_VERSION:1.2.0  ,1.2.1  1.5.0
CUDNN: NO

when I train resnet101 ,if i set export MXNET_CPU_WORKER_NTHREADS=24, I'll get this mistake.
If I don't  set  MXNET_CPU_WORKER_NTHREADS or MXNET_ENGINE_TYPE，it work good.
If I change it to something else backbone ，it work good.
if i change CUDA to 10.0,it work good.

**Comment by szha** [comment]
@olojuwin thanks for reporting. could you try cuda 9.2 for the latest version as well? also, it would be helpful if you could clarify which script you used for training resnet101

**Comment by olojuwin** [comment]
@szha ok，I will test later。
I'll give you the script first：https://github.com/deepinsight/insightface/tree/master/recognition

**Comment by olojuwin** [comment]
@szha 
#!/bin/bash
#source /etc/profile
export MXNET_CPU_WORKER_NTHREADS=2
#export MXNET_CUDNN_AUTOTUNE_DEFAULT=1
export MXNET_ENGINE_TYPE=ThreadedEnginePerDevice
#export MXNET_ENGINE_TYPE=NaiveEngine
#export MXNET_ENGINE_TYPE=ThreadedEngine
#export MXNET_ENABLE_GPU_P2P=0

#MXNET_GPU_COPY_NTHREADS=6
#MXNET_EXEC_NUM_TEMP=4
#MXNET_GPU_WORKER_NTHREADS=4


#DATA_DIR=path/face_emore
NETWORK=r100
JOB=Insightface
#MODELDIR="../model-$NETWORK-$JOB-faces_emore"
#mkdir -p "$MODELDIR"
#PREFIX="$MODELDIR/model"
LOGFILE="log/log-$NETWORK-`date +%Y-%m-%d-%H-%M`.log"
LOSSSET=combined
#LOSSSET=arcface
#DATASET=glint
DATASET=retina


CUDA_VISIBLE_DEVICES='0,1,2,3' python -u train.py --network "$NETWORK" --loss "$LOSSSET" --dataset "$DATASET" --per-batch-size 56  --pretrained ./models/r100-combined-retina/model --pretrained-epoch 31 --lr 0.1 2>&1 | tee $LOGFILE

**Comment by ddavydenko** [comment]
Is there any particular reason you are not including cuDNN even though you are using CUDA? I suggest you compile with cuDNN enabled and try CUDA 9.2. Also, please refrain from using 1.2 version as it is no longer supported.

**Comment by olojuwin** [comment]
@ddavydenko 
Because 1.2.1 is much faster than other versions.
I use mxnet's default cudnn，and I've tried to compile with cudnn，but the same thing happens.

**Comment by samskalicky** [comment]
@olojuwin as requested by @szha please try CUDA 9.2 with the latest version (1.5.1)
